q2_diversity/tests/test_alpha.py

```diff
@@ -105,6 +108,54 @@ def test_alpha_phylogenetic_empty_table(self):
         with self.assertRaisesRegex(ValueError, "empty"):
             alpha_phylogenetic(table=t, phylogeny=tree, metric='faith_pd')
 
+    def test_alpha_phylogenetic_alt(self):
+        table = self.get_data_path('two_feature_table.biom')
+        tree = self.get_data_path('three_feature.tree')
+        actual = alpha_phylogenetic_alt(table=table,
+                                        phylogeny=tree,
+                                        metric='faith_pd')
+        # expected computed with skbio.diversity.alpha_diversity
+        expected = pd.Series({'S1': 0.75, 'S2': 1.0, 'S3': 1.0},
+                             name='faith_pd')
+        pdt.assert_series_equal(actual, expected)
+
+    def test_alpha_phylogenetic_alt_non_phylo_metric(self):
+        table = self.get_data_path('two_feature_table.biom')
+        tree = self.get_data_path('three_feature.tree')
+        with self.assertRaises(ValueError):
```

Comment:
Can you use assertRaisesRegex to check the error message?

Reply:
done! I also added checks to the error messages in the existing tests for `alpha` and `alpha_phylogenetic`